.github/workflows/release.sh

```diff
@@ -106,37 +113,47 @@ echo "::endgroup::"
 
 pick_otp_vsn() {
     global_OTP_VSN=undefined
+    local oldest_supported=undefined
```

Comment:
We're thus able to exit earlier, below, if this variable is not set to something other than `undefined.